proto/api/v1/vuln_mgmt_workload_service.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package v1;
+
+import weak "google/api/annotations.proto";
+import "storage/deployment.proto";
+import "storage/image.proto";
+
+option go_package = "./api/v1;v1";
+option java_package = "io.stackrox.proto.api.v1";
+
+message VulnMgmtExportWorkloadsRequest {
+  int32 timeout = 1;
+  string query = 2;
+}
+
+// The workloads response contains the full image details including the
+// vulnerability data.
+message VulnMgmtExportWorkloadsResponse {
+  storage.Deployment deployment = 1;
+  repeated storage.Image images = 2;
+}
+
+// VulnMgmtWorkloadService APIs are used to manage workload vulnerabilities.
+service VulnMgmtWorkloadService {
```

Comment:
Not sure whether this should be `VulnMgmtWorkloadService` or just `VulnMgmtService` - right now it only applies to the workloads, but it may be in the future that also _node_ vulnerabilities will be interesting (or any other arbitrary thing that's platform related), seems like the better choice to name it just `VulnMgmtService`.